tasks/loop-mocha.js

```diff
@@ -33,7 +33,7 @@ module.exports = function (grunt) {
       otherOptions = {},
       reportLocation = loopOptions.reportLocation || '',
       asyncMethod = (loopOptions.parallel && loopOptions.parallel !== false) ? "map" : "mapSeries",
-      binPath = '.bin/mocha' + (process.platform === 'win32' ? '.cmd' : ''),
+      binPath = options.mocha_bin || '.bin/mocha' + (process.platform === 'win32' ? '.cmd' : ''),
```

Comment:
Sorry I know I suggested only adding ".cmd" in the default case (no options.mocha_bin provided), but likely it is the opposite that is more helpful. I.e. if a project defines a custom mocha path:

/our/path/to/mocha

Then in order for that to be compatible on both *nix and win developer machines, we'd need to conditionally add ".cmd" to the custom value as well.